Source/LoadByOS/WindowsConfigApp/WindowsConfigApp.csproj

```diff
@@ -2,24 +2,15 @@
 
   <PropertyGroup>
     <OutputType>WinExe</OutputType>
-    <TargetFramework>net7.0-windows</TargetFramework>
+    <TargetFramework>net7.0</TargetFramework>
     <EnableWindowsTargeting>true</EnableWindowsTargeting>
-    <UseWindowsForms>true</UseWindowsForms>
```

Comment:
This is interesting. One of the reasons these were created is I thought this config was necessary for the windows one.

Reply:
Makes sense. I really like the LoadByOS architecture. If we need to make it winforms again, we can do so again just as effortlessly without affecting the other OSs.

Reply:
That's my thought too. It also cleans up avalonia by abstracting all of that IsOS checks away.